fury/shaders/billboard_dec.vert

```diff
@@ -1,4 +1,5 @@
 /* Billboard  vertex shader declaration */
 in vec3 center;
+
 out vec3 centeredVertexMC;
```

Comment:
I'd like to see these follow the VTK naming convention. So this would be `centeredVertexMCVSOutput`.